rome_app/models.py

```diff
@@ -72,6 +72,29 @@ class Essay(models.Model):
     pids = models.CharField(max_length=254, null=True, blank=True, help_text='Comma-separated list of pids for books or prints associated with this essay.')
     people = models.ManyToManyField(Biography, null=True, blank=True, help_text='List of people associated with this essay.')
 
+    def preview(self):
+        return self.text[:254]
+
+    def related_works(self):
+        num_prints_estimate = 6000
+        if self.pids is None:
+            return {}
+        else:
+            pidlist = ["pid:\"%s:%s\"" % (app_settings.PID_PREFIX, p) for p in self.pids.split(",")]
+            query = "ir_collection_id:621+AND+display:BDR_PUBLIC+AND+(%s)&fl=primary_title,rel_has_pagination_ssim,rel_is_part_of_ssim,creator,pid,genre" % "+OR+".join(pidlist)
```

Comment:
you can use query = self._get_related_works_query() here, instead of these 2 lines.